README.md

```diff
@@ -132,6 +132,7 @@ List of options provided by CMake:
 | UR_BUILD_ADAPTER_CUDA   | Fetch and use cuda adapter from SYCL                   | ON/OFF     | OFF     |
 | UR_BUILD_ADAPTER_HIP    | Fetch and use hip adapter from SYCL                    | ON/OFF     | OFF     |
 | UR_BUILD_ADAPTER_NATIVE_CPU | Fetch and use native-cpu adapter from SYCL         | ON/OFF     | OFF     |
+| UR_BUILD_ADAPTER_ALL    | Fetch and use all supported adapter from SYCL          | ON/OFF     | OFF     |
```

Comment:
unrelated: 

@kbenzie , @pbalcer : why we have here "fetch and use from SYCL"? We build the adapters in UR, and then update the tag in the SYCL repo, so it is actually the SYCL repo the one fetching the changes from the unified-runtime repo.

Reply:
heh, I was thinking about the same thing yesterday... I believe it's here because in the beginning it was the other way around and we just didn't change the description...

Reply:
That's right. We should change it.

Reply:
I've changed the wording here to simply `Build the X adapter`